bonsai/sdk/src/sync/mod.rs

```diff
@@ -0,0 +1,485 @@
+// Copyright 2023 RISC Zero, Inc.
```

Comment:
This file seems like a duplication of the async impl. I would prefer if we could have a single implementation that is driven by two frontends. The `reqwest` blocking has a reasonable example of unifying the core implementation: https://github.com/seanmonstar/reqwest/blob/master/src/blocking/client.rs